docs/javascript/exercises/5.md

```diff
@@ -86,7 +86,7 @@ The only thing we are doing in this file is including the `exercise5` module we
 Bundles can do more as well.
 Then can include initialization code for example.
 
-See Plone's [default bundle](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone.js).
+See Plone's [default bundle](https://github.com/plone/plone.staticresources/blob/6721e8ab1938efec63aa182dcb55a5ef7b2abe4f/src/plone/staticresources/static/plone.js).
```

Comment:
This also appears to have gone away. This sentence should either be removed or updated with its replacement.